README.md

```diff
@@ -152,7 +147,8 @@ If you want to participate in the network by running a Raiden Service Bundle you
    - We would appreciate it if you allow us access to the monitoring interfaces
      (to do that uncomment the default values of the `CIDR_ALLOW_METRICS` and `CIDR_ALLOW_PROXY` settings).
    - We also recommend that you provide your own monitoring. The setup of which is currently out of scope of this document.
-1. Make sure, that the account, configured in `KEYSTORE_FILE`, has enough funding to register as a service operator.
+   - Please, read carefully the disclaimers for the path finding and monitoring service and uncomment the variables `XX_ACCEPT_DISCLAIMER` if you agree. Note, that the services won't start if you do not. 
+1. Make sure, that the account, configured in `KEYSTORE_FILE`, has enough funding to register as a service operator (you might want to have a look at [Registering as a RSB Provider](#installation/registering) below first). Make sure that the account is stored in `${DATA_DIR}/keystore/`.
 1. If you haven't done so before, run `./register-service-provider.sh` (it uses configuration values from `.env`).
 1. Run `docker-compose up -d` to start all services
```

Comment:
I don't see any mentions here of the `error` that some people will run into like I did.

Reply:
You can see the Attention Box "NOTE" field

Reply:
Thanks. For some reason that wasn't  part of the diff I was seeing of this PR